<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FractalsClass</class>
 <widget class="QMainWindow" name="FractalsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fractals</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>555</width>
         <height>278</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="WidgetFractal" name="myLabelImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_image"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionChoose_fractal"/>
   <addaction name="actionConfigure_fractal"/>
   <addaction name="actionZoomPlus"/>
   <addaction name="actionZoomMinus"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_image">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChoose_fractal">
   <property name="text">
    <string>Choose fractal</string>
   </property>
  </action>
  <action name="actionConfigure_fractal">
   <property name="text">
    <string>Configure fractal</string>
   </property>
  </action>
  <action name="actionZoomPlus">
   <property name="text">
    <string>Zoom +</string>
   </property>
  </action>
  <action name="actionZoomMinus">
   <property name="text">
    <string>Zoom -</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetFractal</class>
   <extends>QLabel</extends>
   <header>widgetfractal.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Fractals.qrc"/>
 </resources>
 <connections/>
</ui>
